## parameter for uav_control

## 2020.11.04测试发现飞机起飞有较大超调，未发现原因，但起飞后控制正常

control:
  ## 控制器编号：0 for PX4_ORIGIN, 1 for PID, 2 for UDE, 3 for NE
  controller_type : default
  # 是否支持外部控制
  enable_external_control : false
  ## 起飞高度,建议设置小一点
  Takeoff_height : 0.4
  ## 降落模式选择
  Land_mode: 1
  ## 降落速度
  Land_speed : 0.2
  ##　上锁高度
  Disarm_height : 0.1
  ## 定位源: 0 for MOCAP, 1 for T265, 2 for GAZEBO, 3 for FAKE_ODOM, 4 for GPS, 5 for RTK, 6 for UWB, 7 for VINS, 8 for OpticalFlow
  location_source : 2
  ## 最大安全速度
  maximum_safe_vel_xy : 0.2
  maximum_safe_vel_z : 0.2
  ## 最大vision/px4速度误差
  maximum_vel_error_for_vision : 100.0

##　地理围栏，根据实际情况设置
geo_fence:
  x_min: -2.0
  x_max: 6.0
  y_min: -3.0
  y_max: 3.0
  z_min: -0.3
  z_max: 1.0
